src/layers/bitmask-layer-shaders.js

```diff
@@ -0,0 +1,67 @@
+export const vs = `
+#define SHADER_NAME xr-layer-vertex-shader
```

Comment:
Does this shader name need to be more specific? Not sure if a conflict would ever cause a bug, or if it is just used for debugging?

Reply:
Oh good point.  I don't think there's an issue since the shaders are internal to the `Model` of the layer, I think, but I will definitely change this